scripts/vote.ts

```diff
@@ -77,8 +76,13 @@ const app = command({
       displayName: "vote_weight",
       description: "Voting weight"
     }),
+    keyfile: positional({
```

Comment:
I think we can remove `my_id` now.  IIRC it was only used to select a hosted account.